client/src/modules/MentorRegistry/components/MentorRegistryModal.tsx

```diff
@@ -0,0 +1,115 @@
+import { Col, Form, Modal, Select, Spin, Typography } from 'antd';
+import React, { useCallback } from 'react';
+import { ExclamationCircleOutlined } from '@ant-design/icons';
+import { Course } from 'services/models';
+
+interface MentorRegistryModalProps {
+  modalData: any;
+  modalLoading?: boolean;
+  courses: Course[];
+  handleModalSubmit: (arg: any) => void;
+  getInitialValues: (arg: any) => any;
+  resendConfirmation: (record: any) => void;
+  onCancel: () => void;
+  cancelMentor: (githubId: string) => Promise<void | undefined>;
```

Comment:
could it be just one `cancelMentor`, which combines logic for these 2 same functions?

Reply:
I'm afraid not. This two functions have different logic and purpose. One of them cancel mentor, other one - closes modal. If I made it in one function it would be overloaded

Reply:
oh, it see. I confused due to naming
onCancel & cancelMentor. while it is `onClose` vs `canceMentor`

Reply:
refactoring existing code is not a part of this task, this two functions were already existed